experimental/auth-logic/provenance_build_wrapper_test.go

```diff
@@ -22,17 +22,22 @@ import (
 	"github.com/project-oak/transparent-release/slsa"
 )
 
-func (p provenanceBuildWrapper) Identify() Principal {
+func (p provenanceBuildWrapper) identify() (Principal, error) {
 	provenance, provenanceErr := slsa.ParseProvenanceFile(p.provenanceFilePath)
 	if provenanceErr != nil {
-		panic(provenanceErr)
+		return Principal{}, provenanceErr
 	}
 
 	applicationName := provenance.Subject[0].Name
-	return Principal{fmt.Sprintf("\"%v::ProvenanceBuilder\"", applicationName)}
+	return Principal{fmt.Sprintf(`"%v::ProvenanceBuilder"`, applicationName)}, nil
 }
 
 func TestProvenanceBuildWrapper(t *testing.T) {
+	handleErr := func(err error) {
+		if err != nil {
+			panic(err)
```

Comment:
What is the different between `panic` and `t.Fatal`? Should not `t.Fatal` or `t.Fatalf` be used with a clear error message? Or even `t.Errorf`, if it makes sense to have the test "keep going"?

Also I think it is more conventional to have inline error handling, even though you'd end up with code that is mostly doing error handling.

Reply:
OK. I made it `t.Fatal` and added more info. I don't think these are errors where it makes sense to continue since the next step usually can't happen if there was an error.